linchpin/README.md

```diff
@@ -125,3 +125,10 @@ Examples
 --------
 
 [examples](examples)
+
+Hints
+------
+
+Linchpin contains lot of ansible code and some of those have hidden secrets which makes failed output *censured*. That is great to avoid leaking passwords and others but it is really bad if you need to find out what just happens on the remote machine. To solve this situation you can enable [Ansible debug](https://docs.ansible.com/ansible/latest/reference_appendices/config.html?highlight=ansible_debug#envvar-ANSIBLE_DEBUG) mode.
```

Comment:
I'd prefer something shorter, like:

"For more verbose *logging* from ansible modules used by linchpin you can enable [Ansible debug](https://docs.ansible.com/ansible/latest/reference_appendices/config.html?highlight=ansible_debug#envvar-ANSIBLE_DEBUG) mode."

Reply:
Changed to smaller version with keeping the censured part.

Reply:
Looks okay, just use *censored* instead of *censured*.

Reply:
True, fixed.